core/bin/zksync_server/src/main.rs

```diff
@@ -186,10 +187,17 @@ async fn main() -> anyhow::Result<()> {
     };
 
     // Run core actors.
-    let (core_task_handles, stop_sender, health_check_handle) =
-        initialize_components(&configs, &components, &secrets)
-            .await
-            .context("Unable to start Core actors")?;
+    let (core_task_handles, stop_sender, health_check_handle) = initialize_components(
```

Comment:
Lets use a named fields struct and destruct it here; tuples with >= fields are easy to mess up.

Reply:
that's how it was before.. don't want to expand the scope of pr